logdevice/common/test/NodesConfigurationPollerTest.cpp

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+#include "logdevice/common/NodesConfigurationPoller.h"
+
+#include <algorithm>
+
+#include <folly/Random.h>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "logdevice/common/ObjectPoller.h"
+#include "logdevice/common/RandomNodeSelector.h"
+#include "logdevice/common/configuration/Configuration.h"
+#include "logdevice/common/configuration/nodes/NodesConfigurationCodec.h"
+#include "logdevice/common/test/TestUtil.h"
+#include "logdevice/common/toString.h"
+
+using namespace facebook::logdevice;
+
+namespace {
+
+class MockNodesConfigurationPoller : public NodesConfigurationPoller {
+ public:
+  virtual ~MockNodesConfigurationPoller() {}
+
+  MockNodesConfigurationPoller(
```

Comment:
Given that it's exactly the same constructor of the base clase, I think you can do `using NodesConfigurationPoller:: NodesConfigurationPoller;`. Check the "inheriting constructor" section in https://en.cppreference.com/w/cpp/language/using_declaration.